crypto/txscript/src/lib.rs

```diff
@@ -81,14 +81,69 @@ enum ScriptSource<'a, T: VerifiableTransaction> {
 /// Unlike the static counting approach which counts all possible signature operations,
 /// this tracks only the actually executed signature operations, leading to more accurate
 /// mass calculations and potentially lower fees for conditional scripts.
-#[derive(Debug, Clone, Copy)]
+#[derive(Debug, Clone)]
 pub struct RuntimeSigOpCounter {
     /// Maximum number of signature operations allowed for this input
     pub sig_op_limit: u8,
     /// Remaining signature operations that can be executed
     pub sig_op_remaining: u8,
```

Comment:
Maybe it shouldn't be `pub` anymore?

Reply:
d23e817a777f25762696b75a7ecdff1762862369